RecoLocalCalo/HcalRecProducers/src/HBHEPhase1Reconstructor.cc

```diff
@@ -518,17 +528,17 @@ void HBHEPhase1Reconstructor::processData(const Collection& coll,
             const float t = getTDCTimeFromSample(s);
             const float dfc = getDifferentialChargeGain(*channelCoder, *shape, adc,
                                                         capid, channelInfo->hasTimeInfo());
-            channelInfo->setSample(ts, adc, dfc, rawCharge,
+            channelInfo->setSample(ts-shiftOneTS, adc, dfc, rawCharge,
```

Comment:
It may be more readable to define an ```inputTS``` and ```fitTS``` (or a similar name) to distinguish the two kinds of TSs.

Reply:
I updated the code with `inputTS` and `fitTS` following your suggestion.